### ISSUE TITLE
Confusing Speech Output Editing text in Frames

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32170]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

When reading text to edit in a text frame such as title, composer, additional lyrics etc the screen reader feedback is very confusing.  It repeats letters at the end of lines and skips them on the start of the next line and will not navigate as expected.

STEPS:
1.  Add a text frame and copy the following lines of text into it including line breaks: 
This is a text frame.
Line 2 of the frame
Now on line 3.
Let's see how it reads.
2.  Arrow down the lines of text and hear that the screen reader leaves out 1 or two letters of the first word of each line and wraps the first letter of the next line instead.  So for example "ine 2 of the text frame N"
3. Now use arrows to move along the first line character by character to try to confirm the line actually says what you want it to say and is formatted correctly.  Notice that it does not announce the line break but instead repeats the L of the next line twice.
4.  Now use Ctrl with arrows to move word by word.  The first line reads correctly, but gets stuck at the end repeating the word frame with each subsequent press of the right arrow.
5. Press down arrow (again noting it misses saying the L of line), press home to go to the start of that line and try moving along with Ctrl arrow word by word.  I have had differing results on doing this, but always some sort of gibberish giving the end of one word and the start of the next.

This makes editing text really difficult, especially to know if it is formatted correctly.  It even makes reading text frames difficult.

Related Issues:
[19933](https://github.com/MuseScore/MuseScore/issues/19933)  Speech reads last character when at end of string 
[23013](https://github.com/musescore/MuseScore/issues/23013)  Text does not wrap in text frames

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by avvvvve** [comment]
@zacjansheski or @DmitryArefiev could you please capture a video with audio of this behavior?
